src/EditorFeatures/TestUtilities/EditAndContinue/RudeEditDiagnosticDescription.cs

```diff
@@ -56,5 +56,19 @@ public override string ToString()
 
             return $"Diagnostic(RudeEditKind.{_rudeEditKind}, {arguments}){withLine}";
         }
+
+        internal void VerifyMessageFormat()
```

Comment:
Do you know why the tests didn't hit the compiler path I modified in https://github.com/dotnet/roslyn/pull/59419 ?

Reply:
They just don't exercise that path. The enclosed tests are pretty focused unit style tests, rather than functional tests like the refactoring and analyser tests, which exercise a whole pipeline.